scripts/python/add_users_everyone_github_team.py

```diff
@@ -22,31 +23,7 @@ def print_stack_trace(message):
         del exc_info
 
 
-if len(sys.argv) == 2:
```

Comment:
removed everything from the global context, instead I've moved it into a main method (which seems to be the python convention) - and pass through any variables we need through the function parameters

Reply:
Yeah that is cool, is okay in this script as not too many layers / nesting of functions to pass the parameters along, in the larger script I think this would be hardwork prob best bet at that point is to to turn those one into OOP based instead